package.json

```diff
@@ -15,7 +15,7 @@
     "@types/jest": "24.0.18",
     "@types/lodash": "^4.14.151",
     "@types/node": "^13.9.8",
-    "@types/react": "16.9.3",
+    "@types/react": "16.9.12",
```

Comment:
I had to update this type library, because the older one didn't seem to include the general attribute `translate`, which resulted in that attribute being required for many HDS components. I assume that HDS's npm package was built with a newer version of this type library that does include the `translate` attribute.